hw/ip/padctrl/data/padctrl_fpv_testplan.hjson

```diff
@@ -0,0 +1,167 @@
+{
+  name: "padctrl"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: MioWarl_A
+      desc: '''In either generic or Xilinx mode, if a muxed IO's pad attribute is written through
+            the TLUL interface via the mio_pads registers, this assertion checks if the
+            corresponding mio_attr_o's reserved bits remain 0. This assertion intends to test the
+            right_any_read_legal access policy for these mio_pads registers.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: MioAttr_A
+      desc: '''In either generic or Xilinx mode, if a muxed IO's pad attribute is written through
+            the TLUL interface via mio_pads registers, this assertion checks if the corresponding
+            mio_attr_o value is updated correctly.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: MioBackwardCheck_A
+      desc: '''If the output mio_attr_o has changed, then in the previous clock cycle, there must be a
+            valid write to the mio_pads register or the write enable signal raises.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: DioWarl_A
+      desc: '''In either generic or Xilinx mode, if a dedic'ed IO's pad attribute is written through
+            the TLUL interface via the dio_pads registers, this assertion checks if the
+            corresponding dio_attr_o's reserved bits remain 0. This assertion intends to test the
+            right_any_read_legal access policy for these dio_pads registers.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: DioAttr_A
+      desc: '''In either generic or Xilinx mode, if a dedicated IO's pad attribute is written through
+            the TLUL interface via dio_pads registers, this assertion checks if the corresponding
+            dio_attr_o value is updated correctly.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: DioBackwardCheck_A
+      desc: '''If the output dio_attr_o has changed, then in the previous clock cycle, there must be a
+            valid write to the dio_pads register or the write enable signal raises.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: Clk_A
+      desc: "Check that clk_pad_i is correctly connected to clk_o."
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: Rstpad_A
```

Comment:
RTL seems to have a typo here (`Rstpad__A` with double underscore), can you fix that? 

Also, I think we should rename both the `Clk_A` and `Rstpad_A` to something more meaningful like `ClkIn_A`, `RstIn_A` or `ClkConnectivity_A`, `RstConnectivity_A`... WDYT?

Reply:
nevermind the typo comment, I looked at an outdated version of the RTL.

Reply:
Actually these were two comments :). Can we align the names here to something like `RstInput_A` and `ClkInput_A`? both assertions check the same sort of property...

Reply:
Oh i am sorry that i missed it.... Will update that and also the RTL.